api/src/main/java/org/pmiops/workbench/notebooks/LeonardoNotebooksClientImpl.java

```diff
@@ -143,10 +159,13 @@ public void createRuntime(
     leonardoRetryHandler.run(
         (context) -> {
           runtimesApi.createRuntime(
-              googleProject,
-              runtimeName,
+              runtime.getGoogleProject(),
+              runtime.getRuntimeName(),
```

Comment:
The client does not get to control this. We should continue to set this according to our own logic

Reply:
It's a little hacky but the behavior is still the same because I override the client's values at the beginning of `RuntimeController.createRuntime`. I can move the overriding here.

Reply:
Ah, sorry - I missed that. This is fine